README.md

```diff
@@ -122,21 +120,21 @@ In your application's `Info.plist` file, register your iOS Bundle Identifer as a
         <string>auth0</string>
         <key>CFBundleURLSchemes</key>
         <array>
-            <string>$(YOUR_BUNDLE_IDENTIFIER)</string>
+            <string>YOUR_BUNDLE_IDENTIFIER</string>
         </array>
     </dict>
 </array>
 ```
 
 > If your `Info.plist` is not shown in this format, you can **Right Click** on `Info.plist` in Xcode and then select **Open As / Source Code**.
 
-Finally, go to your [Auth0 Dashboard](${manage_url}/#/applications/${account.clientId}/settings) and make sure that **Allowed Callback URLs** contains the following entry:
+Finally, go to your [Auth0 Dashboard](https://manage.auth0.com) and make sure that your application's **Allowed Callback URLs** field contains the following entry:
 
 ```text
-{YOUR_BUNDLE_IDENTIFIER}://${YOUR_AUTH0_DOMAIN}/ios/{YOUR_BUNDLE_IDENTIFIER}/callback
+YOUR_BUNDLE_IDENTIFIER://YOUR_AUTH0_DOMAIN/ios/YOUR_BUNDLE_IDENTIFIER/callback
```

Comment:
This the right syntax?

Reply:
Yeah, no need for the extra curlys, and the $ bit makes less sense. You can see a practical example below @ https://github.com/auth0/Auth0.swift/pull/303/files/e46ed9886da18f8ddfd4279ed84508466a16c9da#diff-04c6e90faac2675aa89e2176d2eec7d8R140